aspnetcore/fundamentals/hosting.md

```diff
@@ -94,23 +95,23 @@ This setting controls the capture of startup errors.
 
 When `false`, errors during startup result in the host exiting. When `true`, the host captures exceptions during startup and attempts to start the server.
 
-# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+#### [ASP.NET Core 2.x](#tab/aspnetcore2x)
 
 ```csharp
 WebHost.CreateDefaultBuilder(args)
     .CaptureStartupErrors(true)
     ...
 ```
 
-# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+#### [ASP.NET Core 1.x](#tab/aspnetcore1x)
 
 ```csharp
 var host = new WebHostBuilder()
     .CaptureStartupErrors(true)
     ...
 ```
 
----
+* * *
```

Comment:
The tab specification doc calls for 3 consecutive dashes. Let's stick with that.

Reply:
I just saw these start to change somewhere ... I thought they were going for a new format ... along with the four hash tabs.

Where did I see this coming in? I'll research and see where I got this.

Reply:
I think it was this one ... https://github.com/aspnet/Docs/pull/5878

Reply:
These were changed in a lot of spots (but not everywhere).